### PR TITLE
Add custom sortable transform until it is fixed in Patternfly

### PR DESCRIPTION
**Pull request by jtomasek**
This is a temporary workaround for https://github.com/patternfly/patternfly-react/issues/4251
until https://github.com/patternfly/patternfly-react/pull/4252 is merged

### CODE REVIEW

**Comment by jtomasek** [comment]
Do not merge, styles are broken due to another PF issue
Screenshot 2020-05-14 at 13 49 17

**Comment by jtomasek** [comment]
Done. Ready for review: 
Screenshot 2020-05-14 at 14 48 29

File: src/components/ui/table/sortable.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { SortColumn, SortByDirection, IExtra, ITransform } from '@patternfly/react-table';
+import * as React from 'react';
```
Comment (mareklibra) [inline src/components/ui/table/sortable.tsx:3]: nit: flip

File: src/components/clusterConfiguration/HostsTable.tsx [hunk 2]
```diff
@@ -41,7 +41,12 @@ type OpenRows = {
 };
 
 const columns = [
-  { title: 'ID', cellFormatters: [expandable], transforms: [sortable] },
+  {
+    title: 'ID',
+    cellFormatters: [expandable],
+    transforms: [sortable],
+    props: { type: 'button', whf: 'hello' },
```
Comment (mareklibra) [inline src/components/clusterConfiguration/HostsTable.tsx:48]: hello?

**Pull request merged**